ckan/lib/jobs.py

```diff
@@ -271,13 +271,19 @@ def execute_job(self, job, *args, **kwargs):
 
         # The original implementation performs the actual fork
         queue = remove_queue_name_prefix(job.origin)
-        log.info(u'Worker {} starts job {} from queue "{}"'.format(
-                 self.key, job.id, queue))
+
+        if job.func_name:
+            function_name = job.func_name.split('.')[-1]
```

Comment:
`job.func_name` is stored with the full module path, eg `ckanext.myext.jobs.process_data_fields`